bindings_node/src/groups.rs

```diff
@@ -0,0 +1,403 @@
+use std::{ops::Deref, sync::Arc};
+
+use napi::{
+  bindgen_prelude::{Error, Result, Uint8Array},
+  threadsafe_function::{ErrorStrategy, ThreadsafeFunction, ThreadsafeFunctionCallMode},
+  JsFunction,
+};
+use xmtp_cryptography::signature::ed25519_public_key_to_address;
+use xmtp_mls::groups::{
+  group_metadata::{ConversationType, GroupMetadata},
+  group_permissions::GroupMutablePermissions,
+  MlsGroup, PreconfiguredPolicies,
+};
+use xmtp_proto::xmtp::mls::message_contents::EncodedContent;
+
+use crate::{
+  encoded_content::NapiEncodedContent,
+  messages::{NapiListMessagesOptions, NapiMessage},
+  mls_client::NapiXmtpClient,
+  streams::NapiStreamCloser,
+};
+
+use prost::Message;
+
+use napi_derive::napi;
+
+#[napi]
+pub enum GroupPermissions {
+  EveryoneIsAdmin,
+  GroupCreatorIsAdmin,
+}
+
+impl From<PreconfiguredPolicies> for GroupPermissions {
+  fn from(policy: PreconfiguredPolicies) -> Self {
+    match policy {
+      PreconfiguredPolicies::AllMembers => GroupPermissions::EveryoneIsAdmin,
+      PreconfiguredPolicies::AdminsOnly => GroupPermissions::GroupCreatorIsAdmin,
+    }
+  }
+}
+
+impl Into<PreconfiguredPolicies> for GroupPermissions {
+  fn into(self) -> PreconfiguredPolicies {
+    match self {
+      GroupPermissions::EveryoneIsAdmin => PreconfiguredPolicies::AllMembers,
+      GroupPermissions::GroupCreatorIsAdmin => PreconfiguredPolicies::AdminsOnly,
+    }
+  }
+}
+
+#[napi]
+pub struct NapiGroupMetadata {
+  inner: GroupMetadata,
+}
+
+#[napi]
+impl NapiGroupMetadata {
+  #[napi]
+  pub fn creator_inbox_id(&self) -> String {
```

Comment:
this function doesn't return an Inbox ID, just an empty string. thoughts?

Reply:
saw the same, here is an issue with test for reproducing - https://github.com/xmtp/libxmtp/issues/788